app/uk/gov/hmrc/agentclientrelationships/repository/DeleteRecordRepository.scala

```diff
@@ -46,10 +45,10 @@ case class DeleteRecord(
   arn: String,
   clientIdentifier: String,
   clientIdentifierType: String,
-  dateTime: DateTime = now(UTC),
+  zonedDateTime: ZonedDateTime = now(UTC),
```

Comment:
@LGoldfinch2 This change of field name from `dateTime` to `zonedDateTime` looks like it will have affected the (de)serialisation of `DeleteRecord` down on line 84 (`implicit val formats: Format[DeleteRecord] = format[DeleteRecord]`)
We should keep the field name the same so we can continue to read old records of this type from the DB.